pkg/ee/group-project-binding/handler/handler.go

```diff
@@ -136,3 +170,211 @@ func GetGroupProjectBinding(ctx context.Context, request interface{},
 		Role:      binding.Spec.Role,
 	}, nil
 }
+
+// swagger:parameters createGroupProjectBinding
+type createGroupProjectBindingReq struct {
+	common.ProjectReq
+
+	// in: body
+	// required: true
+	Body groupProjectBindingBody
+}
+
+func DecodeCreateGroupProjectBindingReq(c context.Context, r *http.Request) (interface{}, error) {
+	var req createGroupProjectBindingReq
+
+	projectReq, err := common.DecodeProjectRequest(c, r)
+	if err != nil {
+		return nil, err
+	}
+	req.ProjectReq = projectReq.(common.ProjectReq)
+
+	if err := json.NewDecoder(r.Body).Decode(&req.Body); err != nil {
+		return nil, err
+	}
+	return req, nil
+}
+
+func (r createGroupProjectBindingReq) Validate() error {
+	if r.Body.Group == "" {
+		return utilerrors.NewBadRequest("`group` cannot be empty")
+	}
+
+	if err := validateRole(r.Body.Role); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func CreateGroupProjectBinding(ctx context.Context, request interface{},
+	userInfoGetter provider.UserInfoGetter,
+	projectProvider provider.ProjectProvider,
+	privilegedProjectProvider provider.PrivilegedProjectProvider,
+	bindingProvider provider.GroupProjectBindingProvider,
+) error {
+	req, ok := request.(createGroupProjectBindingReq)
+	if !ok {
+		return utilerrors.NewBadRequest("invalid request")
+	}
+
+	err := req.Validate()
+	if err != nil {
+		return utilerrors.NewBadRequest(err.Error())
+	}
+
+	kubermaticProject, err := common.GetProject(ctx, userInfoGetter, projectProvider, privilegedProjectProvider, req.ProjectID, nil)
+	if err != nil {
+		return common.KubernetesErrorToHTTPError(err)
+	}
+
+	userInfo, err := userInfoGetter(ctx, kubermaticProject.Name)
+	if err != nil {
+		return err
+	}
+
+	if err := bindingProvider.Create(ctx, userInfo, &kubermaticv1.GroupProjectBinding{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: fmt.Sprintf("%s-%s", kubermaticProject.Name, rand.String(10)),
+		},
+		Spec: kubermaticv1.GroupProjectBindingSpec{
+			ProjectID: req.ProjectID,
+			Group:     req.Body.Group,
+			Role:      req.Body.Role,
+		},
+	}); err != nil {
+		return common.KubernetesErrorToHTTPError(err)
+	}
+	return nil
```

Comment:
Usually in KKP we return the created/patched resources, could you do that as well to keep consistency?